src/option_parser.c

```diff
@@ -187,20 +187,55 @@ int ini_get_bool(char *section, char *key, int def)
         }
 }
 
-char *clean_value(char *value)
+char *clean_value(char *value, int line_num)
```

Comment:
This entire function is way too complicated at a whopping 6 levels of indentation(!!!) for something that can probably be done in a few lines in the `load_ini_file` function.
What's more is that you assume that the first quote is always the start of the string, this means that something like `String \"with quotes\"` fails to parse. A better approach would be to check the first non-whitespace character of the value, if it's a quote then we can go hunting for the closing quote(preferably in a different function), if not we just unescape all quotes and return.